### PR TITLE
inject command in manifests, bump to 1.5.4

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @openshift/openshift-team-auth-maintainers

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ BUNDLE_IMAGE_TAG?=latest
 
 TEST_OPERATOR_NAMESPACE?=openshift-cert-manager-operator
 
-MANIFEST_SOURCE = https://github.com/jetstack/cert-manager/releases/download/v1.4.0/cert-manager.yaml
+MANIFEST_SOURCE = https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
```
Comment (stlaz) [inline Makefile:16]: commit message and PR title say 1.5.4

File: hack/update-cert-manager-manifests.sh [hunk 2]
```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
```
Comment (stlaz) [inline hack/update-cert-manager-manifests.sh:1]: make the bash script fail on errors (`set -e`)

File: jsonnet/main.jsonnet [hunk 3]
```diff
@@ -40,6 +40,33 @@ local cleanupDeployment(manifest) =
...
+          'app.kubernetes.io/component' in item.metadata.labels
+  then 'cert-manager-deployment/' + item.metadata.labels['app.kubernetes.io/component'] + '/' + item.metadata.name + '-' + suffix[item.kind] + '.yaml'
+  // else, leave it at the top-level
+  else 'cert-manager-deployment/' + item.metadata.name + '-' + suffix[item.kind] + '.yaml';
+
 // top level function (aka 'main')
-function(manifest)
-  cleanupDeployment(cleanupAll(manifest))
+function(manifest) {
+  [std.strReplace(path(item), ':', '-')]: cleanupDeployments(cleanupHelmLabels(item))
```
Comment (stlaz) [inline jsonnet/main.jsonnet:70]: should we remove the help labels on all possible objects, not just deployments?

**Review by stlaz — CHANGES_REQUESTED** [review]
I think we should also add bindata verification to the makefile's `verify` target so that PRs that don't update the bindata when they should get flagged.

File: hack/update-cert-manager-manifests.sh [hunk 4]
```diff
@@ -1,37 +1,51 @@
...
 
+if [ ! -f ./_output/tools/bin/yq ]; then
+    mkdir -p ./_output/tools/bin
+    curl -s -f -L https://github.com/mikefarah/yq/releases/download/v4.13.3/yq_$(go env GOHOSTOS)_$(go env GOHOSTARCH) -o ./_output/tools/bin/yq
+    chmod +x ./_output/tools/bin/yq
+fi
+
```
Comment (stlaz) [inline hack/update-cert-manager-manifests.sh:8]: move below the sourcing

Comment (s-urbaniak) [reply hack/update-cert-manager-manifests.sh:8]: fixed

**Comment by stlaz** [comment]
/hold
for verification

**Pull request merged**